app/crud.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy.orm import Session
+from . import models, schemas
+
+def create_city(db: Session, city: schemas.CityCreate):
```

Comment:
Consider adding error handling for database operations to manage exceptions that might occur during the creation of a city, such as integrity errors or connection issues.